app/models/user.rb

```diff
@@ -80,4 +80,16 @@ def remove_pregnancy(pregnancy)
     pregnancies.delete pregnancy
     reload
   end
+
+  def clear_call_list
+    pregnancies.each do |p|
+      pregnancies.delete(p) if recently_reached?(p)
+    end
+  end
+
+  private
+
+  def recently_reached?(preg)
```

Comment:
I think making them both private methods is right on, since we don't really call them anywhere else, no?